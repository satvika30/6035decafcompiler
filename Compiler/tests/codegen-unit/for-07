class Program {
	void main() {
		for i = 0, 10 {
			if (i > 2) {
				break;
			}
			
			for j = 0, 10 {
				if (j > 3) {
					break;
				}

				callout("printf","%d\n",j);
			}
		}
		
		callout("printf","-\n");
		
		for i = 0, 10 {
			if (i < 7) {
				continue;
			}
			
			for j = 0, 10 {
				if (j < 8) {
					continue;
				}
				
				callout("printf","%d\n",j);
			}
		}
	}
}