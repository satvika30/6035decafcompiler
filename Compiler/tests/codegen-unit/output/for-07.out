0
1
2
3
0
1
2
3
0
1
2
3
-
8
9
8
9
8
9
